website/examples/date-picker-dialog.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { DayPicker } from 'react-day-picker';
+import { usePopper } from 'react-popper';
+
+import { format, isValid, parse } from 'date-fns';
+import FocusTrap from 'focus-trap-react';
+
+export default function Example() {
+  const [selected, setSelected] = React.useState<Date>();
+  const [inputValue, setInputValue] = React.useState<string>('');
+  const [isOpen, isOpenSet] = React.useState(false);
+
+  const refEl = React.useRef<HTMLDivElement>(null);
+  const buttonRef = React.useRef<HTMLButtonElement>(null);
+  const [popperEl, popperElSet] =
+    React.useState<HTMLDivElement | null>(null);
+
+  const closePopper = () => {
+    isOpenSet(false);
+    buttonRef?.current?.focus();
+  };
+
+  const { styles, attributes } = usePopper(
```

Comment:
Better don't remove the "scope" of these generic names. Do not destructure here:

```suggestion
  const popper = usePopper(
```